### PR TITLE
setup.cfg: Build wheel for Python 3.4-3.6 only

### PR DESCRIPTION
**Pull request by jayvdb**
Fixes https://github.com/coala/coala/issues/3383

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -39,7 +39,8 @@ script:
     else
       set -e
       bash .misc/tests.sh
-      python setup.py install
```
Comment (yukiisbored) [inline .travis.yml]: Why is this line removed?

Comment (yukiisbored) [inline .travis.yml]: Oh wait, nvm. I didn't see `pip install ./dist/coala-*.whl`

**Comment by nemaniarjun** [comment]
ack bc8f059293e013e174317a5b9a4fd05ad1ea579a

File: test-requirements.txt [hunk 2]
```diff
@@ -1,3 +1,4 @@
+wheel~=0.29
```
Comment (Mixih) [inline test-requirements.txt]: Alphabetical order please :)

**Comment by Mixih** [comment]
unack bc8f059

**Comment by Mixih** [comment]
Wait we should probably modify circle's config too right @jayvdb ?

**Comment by jayvdb** [comment]
We dont deploy from either CircleCI or TravisCI, so having one use `setup.py install`, and the other build and install a wheel, helps with coverage.

**Comment by Mixih** [comment]
alright

**Comment by Mixih** [comment]
ack 8db4fb7

**Comment by jayvdb** [comment]
@rultor merge
